src/components/molecules/UserList/UserList.tsx

```diff
@@ -29,6 +29,7 @@ interface PropsType {
   isCamp?: boolean;
   attendanceBoost?: number;
   showEvenWhenNoUsers?: boolean;
+  hasTitle?: boolean;
```

Comment:
Props with default values marked as optional makes me ponder if they shouldn't be marked optional. 🤔
No strong opinion.

Reply:
Optional in the types allows us to set the defaults.